1-js/06-advanced-functions/01-recursion/05-output-single-linked-list-reverse/solution.md

```diff
@@ -31,13 +31,13 @@ function printReverseList(list) {
 printReverseList(list);
 ```
 
-# Using a loop
+# با استفاده از حلقه
 
-The loop variant is also a little bit more complicated then the direct output.
```

Comment:
It's a typo, it should've been like this:
>  The loop variant is also a little bit more complicated *than* the direct output.

Made a PR: https://github.com/javascript-tutorial/en.javascript.info/pull/2731